README.md

```diff
@@ -25,7 +25,21 @@ Prompt Palette is meant to be sleek, fast, with an intuitive UI. As useful as po
 Prompt Palette will not ask for your data. Your data is yours.
 
 
-## Tech Used: ![HTML BADGE](https://img.shields.io/static/v1?label=|&message=HTML&color=3c7f5d&style=plastic&logo=html5)![CSS BADGE](https://img.shields.io/static/v1?label=|&message=CSS&color=285f65&style=plastic&logo=css3)![JAVASCRIPT BADGE](https://img.shields.io/static/v1?label=|&message=JAVASCRIPT&color=3c7f5d&style=plastic&logo=javascript)![CHATGPT BADGE](https://img.shields.io/static/v1?label=|&message=CHATGPT&color=285f65&style=plastic&logo=openai)
+
+<h2>How to use:</h2> <h3> Basics:</h3>
+                        <p>Press <strong>cmd + shift + p</strong> to open the extension anywhere in the browser</p>
```

Comment:
We discussed offline that it may be beneficial to tell the user to go to `chrome://extensions/shortcuts` in case the keyboard shortcut doesn't work, or they want to set their own custom key mapping